icecap/docker/setup.sh

```diff
@@ -0,0 +1,20 @@
+# AUTHORS
+#
+# The Veracruz Development Team.
+#
+# COPYRIGHT
+#
+# See the `LICENSE_MIT.markdown` file in the Veracruz root directory for licensing
+# and copyright information.
+
+set -e
+
+if [ ! -f /nix/.installed ]; then
+    echo "Installing Nix..."
+    curl -L https://github.com/nspin/minimally-invasive-nix-installer/raw/dist-11p9fcf3ca/dist/install.sh -o install-nix.sh
```

Comment:
Why not put the content `install.sh` directly here, or do the same as rustup and put it directly into `Dockerfile`?